src/bootstrap/bootstrap.py

```diff
@@ -12,9 +12,25 @@
 import sys
 import tarfile
 import tempfile
+import sqlite3
 
 from time import time
 
+def acquire_lock():
```

Comment:
It'd be great to put a comment on this function, explaining why we're using sqlite to implement this, and the goal of the lock itself.